pkg/kdump/kdump.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if test -f /proc/vmcore; then
+    #
+    # We are in dump capture kernel, nice.
+    #
+    MAX=5
+    DIR=/persist/kcrashes
+
+    # Create a folder
+    mkdir $DIR > /dev/null 2<&1
+
+    # Keep $MAX-1 fresh dumps
+    ls -t $DIR | tail -n +$MAX | xargs --no-run-if-empty -I '{}' rm $DIR/{}
+
+    # Show panic log from the dmesg of a crashed kernel
+    makedumpfile --dump-dmesg /proc/vmcore /tmp/dmesg > /dev/null
+    echo ">>>>>>>>>> Crashed kernel dmesg BEGIN <<<<<<<<<<" > /dev/kmsg
+    sed -n -e '/Kernel panic - not syncing/,$p' /tmp/dmesg | \
+        while read line; do echo $line > /dev/kmsg; done
+    echo ">>>>>>>>>> Crashed kernel dmesg END <<<<<<<<<<" > /dev/kmsg
```

Comment:
Can we also feed this into /persist/reboot-stack as we do in agentlog.RebootStack()?

Don't know if there we can extract the kernel panic string and write that to /persist/reboot-reason as we do in agentlog.RebootReason().

Note that these agentlog functions are today used to handle golang runtime panics, but if we reuse those mechanisms it means we get the logging of the kernel panics from the common code which reads and clears /persist/reboot-stack and /persist/reboot-reason.

Reply:
I can redirect there as well, don't see any problems. I will check that. 
Do I need to redirect to reboot-reason or reboot-stack?

Reply:
reboot-reason is for a single line, so if you have the panic string you can send it there.
reboot-stack is for the stack trace.